<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta property="og:image" content="img/thumbnail.jpg">
    

    <title>www.pythoninpharma.com</title>

    <!-- Bootstrap -->
	  
	  <link href="bootstrap/css/bootstrap.min.css" rel="stylesheet">
	  <link href="pypharma.css" rel="stylesheet">
	  
	  <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
	  
  </head>

  <body data-spy="scroll" data-target="#navbar">

    	<nav class ="navbar navbar-fixed-top navbar-inverse" id="navbar">
	  		<div class="container">
				
				<div class="navbar-header">							
					<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse">
					<span class="sr-only">Toggle navigation</span>
					
					<span class="icon-bar"></span>	
					<span class="icon-bar"></span>
					<span class="icon-bar"></span>
					</button>
				
					<a class="navbar-brand" href="index.html">www.pythoninpharma.com</a>
				</div><!-- navbar-header -->	
			
				<div class="collapse navbar-collapse" id="navbar-collapse">
					<ul class="nav navbar-nav navbar-right">
					<li class="active"><a href="#home">Home</a></li>
					<li><a href="#about-pypharma">About</a></li>		
					<li><a href="#programme">Programme</a></li>
					<li><a href="#location">Location</a></li>		
					<li><a href="#keynotes">Keynotes</a></li>
					<li><a href="#workshops">Workshops</a></li>
					<li><a href="#executive-committee">Executive Committee</a></li>
					<li><a href="#diversity-grants">Diversity Grants</a></li>
					<li><a href="#contact">Contact</a></li>
				 	</ul>
				</div>	
			</div><!-- container -->
	  	
	  	</nav>

   		<div class="cover" id="home">
	  		<div class="cover-text">
				<img src="img/pypharma-logo.png" alt="PyPharma 2019" 
                    class="pypharma-logo"></img>
                <!--<h1>PyPharma 2019</h1>-->
				<p class="medium">NOVEMBER 21-22, 2019, BASEL<p>
                <p class="lead">The first and only conference dedicated to Python in the pharma data science world.</p>
			
	  			<a href="#about-pypharma" role="button" 
                    class="btn btn-danger btn-lg">Learn More</a>
			</div>
	  	</div>

	  	<section class="testimonial">
			<div class="container">
				<blockquote>
					&ldquo; Hiding within those mounds of data is knowledge that could change the life of a patient, or change the world. &rdquo;
					<cite>&mdash; Atul Butte</cite>
				</blockquote>
			</div>
	  	</section>
	  
	  	<section id="about-pypharma">
	  		<h2>About PyPharma</h2>
	  			<div class="container">
					<div class="row">
						<div class="col-sm-4">
							<h3>Purpose</h3>
							<p>PyPharma will be a meeting, exchange and learning point for industry and academic scientific python users in pharma. At PyPharma, we will learn about the most interesting current challenges in the field and the new and exciting python tools and packages that can be used to tackle them.</p>
						</div>
						<div class="col-sm-4">
							<h3>Scope</h3>
							<p>PyPharma will be aimed at pharmaceutical python users of all levels working in data science, and it will welcome workshops, talks and posters covering all aspects of the pharmaceutical lifecycle and ecosystem where python tools are applied as well as data modalities.</p>
						</div>
						<div class="col-sm-4">
							<h3>Concept</h3>
							<p>PyPharma is fully run by volunteers in the pharmaceutical industry and academia and hosted by Roche and the University of Basel. Attendance will be based on invites, and it will be a single track, 2-day conference. Attendance will be purposely kept small to maximize interaction between the audience and the speakers.</p>
						</div>
					
					</div>
				</div>
	  </section>

      <section id="programme">
	  		<h2>Programme</h2>
	  		<div class="container">
				<div class="row">
					<div class="col-sm-12">
                        <h3>Day 1 (November 21st)</h3>
                        <style>
                            thead.thead-dark {
                                color: #fff;
                                background-color: #343a40;
                                border-color: #454d55;
                                vertical-align: middle;
                            }
                        </style>
                        
                        <table class="table table-striped table-bordered">
                        <thead class="thead-dark"><tr><th>Time</th><th>Biozentrum Seminarraum 104</th><th>Kollegienhaus Mehrzweckraum</th><th>Missionsstrasse 64a Computerraum</th><th>Milsionsstrasse 64a Seminarraum</th></tr></thead><tbody>
                         <tr><td>09:00-10:30</td><td>Deep Learning for Natural Language Processing in Pharma and Biomedical Applications</td><td>Building pharmaceutical web applications with Dash</td><td></td><td>Genomic Data Analysis with Pandas</td></tr>
                         <tr><td>10:30-11:00</td><td colspan="4">Coffee break</td></tr>
                         <tr><td>11:00-12:00</td><td>Deep Learning for Natural Language Processing in Pharma and Biomedical Applications</td><td>Building pharmaceutical web applications with Dash</td><td></td><td>Genomic Data Analysis with Pandas</td></tr>
                         <tr><td>12:00-13:00</td><td colspan="4">Lunch</td></tr>
                         <tr><td>13:00-14:30</td><td>Interpretability in Machine Learning for Computational Biology</td><td>Snakemake for reproducible analyses</td><td>Hands-on Bayesian: what, why, how?</td><td>Cheminformatics workshop from QSAR to DNN</td></tr>
                         <tr><td>14:30-15:00</td><td colspan="4">Coffee break</td></tr>
                         <tr><td>15:00-17:00</td><td>Interpretability in Machine Learning for Computational Biology</td><td>Snakemake for reproducible analyses</td><td>Hands-on Bayesian: what, why, how?</td><td>Cheminformatics workshop from QSAR to DNN</td></tr>
                        </tbody>
                        </table>
                        
                        <h3>Day 2 (November 22nd)</h3>
                        <table class="table table-striped table-bordered">
                        <thead class="thead-dark"><tr><th>Time</th><th>Speaker</th><th>Affiliation</th><th>Title</th><th>Description</th></tr></thead><tbody>
                        <tr><td>08:00-08:30</td><td colspan="4">Breakfast</td></tr>
                        <tr><td>08:30-09:00</td><td colspan="4">Conference Opening</td></tr>
                        <tr><td>09:00-09:50</td><td>Keynote 1: Greg Landrum</td><td>T5 Informatics</td><td>Discovering the PyData Ecosystem and RDKit</td><td>To be added soon.</td></tr>
                        <tr><td>09:50-10:20</td><td>Martin Preusse</td><td>University of Freiburg</td><td>Integrating Molecular and Clinical Data with Python Knowledge Graphs & Neo4j</td><td>Data is everywhere but generating useful knowledge is difficult. To create value from data you need solutions for all steps of the process: Data acquisition, cleaning, structuring, annotation, integration, modeling, validation and much more. Even when all technological challenges are addressed, you have to account for the most important aspect of data applications: The human beings who decide what to do and how to use the technology. There are two key aspects of data applications that are overlooked by the current AI hype. First, integrating heterogenous data is still an issue. Second, getting an overview of data is essential to take informed decisions. Knowledge graphs have the potential to adress these aspects and to serve as the central hub of your data application. They link heterogenous data sources and provide access to structured data for decision making, data analysis and end-user applications. In my talk I will give examples how biomedical knowledge graphs implemented in the graph database Neo4j help to link molecular and patient level data. On top of that, I discuss how knowledge graphs can support the use of real world evidence for early stage drug development.</td></tr>
                        <tr><td>10:20-10:40</td><td colspan="4">Poster Session / Coffee Break</td></tr>
                        <tr><td>10:40-11:10</td><td>Flaviu Cipcigan</td><td>IBM Research</td><td>Python for molecular dynamics workflows and conformational sampling of cyclic peptides</td><td><p>Molecular dynamics holds great promise for the pharmaceutical industry. Both targets and ligands are dynamic, exploring multiple conformations. Thus, the typical methods that assume a single binding pose or a single shape for a pocket will have limited ability to predict realistic interactions. As pharmaceutical industry moves to larger drugs such as cyclic peptides or therapeutic proteins, molecular flexibility is starting to be more and more important. For example, cell permeability of cyclic peptides can be modulated by their conformational changes.</p> <p>In order for molecular dynamics to be fully adopted by the industry, better tools are needed to manage computational workflows and explore the resulting data. Here, I will discuss a Python middleware that simplifies the running and analysis of molecular dynamics simulations and allows the user to interactively analyse the resulting multidimensional data. I will also present the use of this tool in the context of a research challenge of predicting the permeability of cyclic peptides.</p></td></tr>
                        <tr><td>11:10-12:00</td><td>Keynote 2: Kai Blin</td><td>Novo Nordisk Foundation, Center for Sustainability</td><td>Mining microbial genomes for interesting metabolites</td><td>Secondary metabolites produced by microorganisms are the main source of bioactive compounds thatare in use as antimicrobial and anticancer drugs, fungicides, herbicides and pesticides. As anincreasing number of microorganisms is becoming resistant against the antimicrobial compoundscurrently in use, there is a dire need for new compounds and compound classes that show newmodes of action to avoid the current resistance mechanisms. In the last decade, the increasingavailability of microbial genomes has established genome mining as a very important method for theidentification of biosynthetic gene clusters (BGCs) responsible for producing such novel compounds.This talk will present antiSMASH, a Python-based tool to mine microbial genomes for interestingBGCs in order to find new bioactive lead compounds. I will show how Python helped antiSMASH tobecome the most widely used tool in the field, with over half a million of processed jobs and over 2000weekly users on the public web service alone. I will also present the up- and downsides of running acomplex analysis pipeline as a web service in Python.</td></tr>
                        <tr><td>12:00-13:00</td><td colspan="4">Lunch / Poster Session</td></tr>
                        <tr><td>13:00-13:50</td><td>Keynote 3: Maria Rodriguez Martinez</td><td>IBM, Zurich, Research Laboratory</td><td>Artificial Intelligence approaches for personalized medicine.</td><td>In recent years, AI has become a very active field in computer science and models with astounding performances in a broad area of applications such as computer vision, speech recognition and natural language processing have been developed. In computational biology, the recent availability of large amounts of data generated by large international consortia together with technical developments facilitating the implementation and training of more performant models have made possible the broad application of deep learning and machine learning approaches to a vast set of problems.</td></tr>
                        <tr><td>13:50-14:20</td><td>Franciska Oschman</td><td>ETH Zurich</td><td>Applications of machine learning in research</td><td>In this talk, I will present current activities at the Computational Systems Biology group in IBM Research, Zurich, that illustrate the application of AI approaches to unravel disease mechanisms and develop personalized patient models. For instance, I will show how models for text ingestion can be used to automatically extract knowledge from biomedical publications and obtain comprehensive maps of molecular interactions. I will also show how multi-modal neural networks can be trained to ingest disparate data types, such as compound molecular structures, transcriptomic data and prior molecular knowledge, to predict drug sensitivity in cancer cell lines. Finally, I will illustrate how deep learning models can be adapted to characterize tumor heterogeneity in single-cell data.</td></tr>
                        <tr><td>14:20-14:50</td><td>David Marcus</td><td>GlaxoSmithKline</td><td>Explore, exploit, and extrapolate: How AI-driven SAR navigation facilitates lead optimisation in drug discovery</td><td>Small molecule drug discovery involves a complex multi-parameter optimisation process with cycles of design, make and test to establish a desired compound profile. Within this context, machine learning methods, experimental design and de-novo structure generation have all found a place to facilitate and accelerate lead optimisation. However, they have tended to be used in a reactive manner, to address problems posed by program teams rather than as a continual and proactive process. In this presentation we will describe how data-driven cheminformatics-based AI methods have the potential to automate parts of the lead optimisation process which historically has been a very time-consuming task. This strategy adds the ability to explore and exploit multiple paths within chemical space and suggest structural modifications that will gain better understanding of the SAR problem at hand. In addition, by continually improving computational models we can extrapolate to new regions of chemical space and suggest novel compounds. The implications of automation for the human-machine interface will be explored and illustrated with examples from BRADSHAW, GSK's experimental automated design environment.</td></tr>
                        <tr><td>14:50-15:10</td><td colspan="4">Poster Session / Coffee Break</td></tr>
                        <tr><td>15:10-15:40</td><td>Damian Roqueiro</td><td>ETH Zurich</td><td><strike>Machine Learning for Deep Electrophysiological Phenotyping of Human Dopaminergic Neurons</td><td>Cell-reprogramming has opened up new and exciting opportunities to study the mechanisms underlying neurological disorders of patient-derived tissue in vitro. These opportunities also pose some challenges:  (i) there are currently no widely-accepted analytical tools and (ii) there are few standardized assays available to assess the functionality of human induced pluripotent stem cell (iPSC)-derived neurons. With the advent of high-density microelectrode arrays (HD-MEAs), researchers have the ability to probe the cellular and network phenotypes of commercially available iPSC-lines. In this talk we will showcase a comparison between midbrain dopaminergic neurons from a healthy subject (WT), and an isogenic cell line carrying the A53T alpha-synuclein mutation associated with early-onset familiar Parkinson's disease (PD). We developed a pipeline in Python to reliably infer features that allow characterizing iPSC neuronal networks at the single-cell and network level, and investigated the development of these features. Machine learning models were applied to infer which features were most informative to predict the phenotype. The talk will cover these models in detail and will provide pointers to the Python resources used to conduct the analyses.</strike></td></tr>
                        <tr><td>15:40-16:30</td><td>Keynote 4: Michał Januszewski</td><td>Google Research</td><td>Reconstruction of neural wiring diagrams from large-scale volume EM data using Python and machine learning</td><td>Dense, synaptic-level mapping of neural circuits requires tracing neurons within volume electron microscopic datasets. With single volumes now reaching the scale of petabytes, manual approaches are no longer viable and automation becomes a necessity. Within the Connectomics team at Google (https://ai.google/research/teams/perception/connectomics/) we have developed infrastructure and algorithms for storage, analysis, and visualization of such large volumetric datasets. Much of this software stack is open source and implemented in Python. In the talk, I will describe how we use this reconstruction pipeline to convert series of EM images into brain connectivity diagrams, and how modern machine learning techniques based on deep neural networks allow us to solve classification and segmentation problems in such datasets.</td></tr>
                        <tr><td>16:30-17:00</td><td colspan="4">Conference Closing</td></tr>
</tbody></table>
                    </div>
                </div>
            </div>
      </section>
            
      <section id="location" >
	  	<h2>Location</h2>
	  	<div class="container">
			<div class="row">
				<div class="col-sm-6">
					<h3>Day 1 (November 21st, 2019)</h3>
					<p>Day 1 will consist of workshops and will take place at the University of Basel. The room locations are:</p>
                    <ul>
                        <li>Kollegienhaus Mehrzweckraum, <a href="https://goo.gl/maps/VaqzqFwV8BNhf8PE6">Petersgraben 50</a></li>
                        <li>Biozentrum Seminarraum 104, <a href="https://goo.gl/maps/poJiWsPHciJyE3FH9">Klingelbergstrasse 70</a></li>
                        <li>Missionsstrasse 64a Seminarraum 00.004, <a href="https://goo.gl/maps/iNzEEpmdAoKaqPyV6">Missionsstrasse 64a</a></li>
                        <li>Missionsstrasse 64a Computerraum 00.010, <a href="https://goo.gl/maps/iNzEEpmdAoKaqPyV6">Missionsstrasse 64a</a></li>
                    </ul>
				</div>
				<div class="col-sm-6">
					<h3>Day 2 (November 22nd, 2019)</h3>
					<p>Day 2 will consist of keynotes and talks and will take place at the <a href="https://goo.gl/maps/YHTTSuPJpaLi5mRc8">Roche Viaduktstrasse amphitheater</a> (close to the Basel SBB station).</p>
				</div>
			</div>
		</div>
	  </section>
	  
	  <section id="keynotes">
		<h2>Keynote Speakers</h2>
		  <div class="container">
			<div class="row">
				
                <div class="col-sm-3">
                  <div class="pypharma-member">
					<img src="img/exec/maria_rodriguez_martinez.png" alt="María Rodríguez Martínez" class="img-circle pypharma-thumbnail">
					<h4>María Rodríguez Martínez</h4>
					<h5>IBM, Zurich Research Laboratory</h5>
                    
					<div id="keynote_1" class="collapse pypharma-bio">
					    <p>María Rodríguez Martínez did her undergraduate studies in Physical Sciences at Universidad Complutense de Madrid. She then did her PhD in Theoretical Cosmology, at the Institut d’Astrophysique de Paris. Her PhD research focused on developing cosmological models of the early evolution of the universe with additional spatial dimensions. After completing her PhD, she moved to the Hebrew University in Jerusalem, where she focused on setting astrophysical bounds to theories that break Lorentz symmetry using the high-energy emissions from Gamma-Ray Bursts, extremely powerful explosions of gamma rays coming from outside our galaxy.

In 2007, she transitioned into the field of Systems Biology as a postdoc at the Weizmann Institute of Science in Rehovot (Israel). Her research was devoted to the development of quantitative descriptions of biological networks and the complex interactions within. In 2009, she moved to Columbia University where she developed quantitative models to understand cancer gene dysregulation.

María joined IBM as a Research Staff Member in November 2013. Her research at IBM focuses on integrating different high-throughput molecular datasets in order to build comprehensive molecular models of disease that can help clinicians to provide better diagnoses and suggest personalized therapies.</p>
                    </div>
                    
                    <button class="btn btn-primary" type="button" 
                        data-toggle="collapse" data-target="#keynote_1" 
                        aria-expanded="false" aria-controls="keynote_1">
                        Show Bio
                    </button>
                  </div>
				</div>

                <div class="col-sm-3">
                  <div class="pypharma-member">
					<img src="img/exec/kai_blin.jpg" alt="Kai Blin" class="img-circle pypharma-thumbnail">
					<h4>Kai Blin</h4>
					<h5>Novo Nordisk Foundation Center for Biosustainability</h5>
                    
					<div id="keynote_2" class="collapse pypharma-bio">
					    <p>After obtaining a degree in bioinformatics from the computer science faculty at the University of Tübingen in 2009, Kai switched to Institute for Microbiology and Infection Medicine at the same university to obtain his PhD. During his PhD project, he co-developed the antiSMASH genome mining tool, initially released in 2010. Building on his software engineering knowledge obtained working on Open Source software projects since his undergrad days, Kai has focused on developing translational bioinformatics tools and databases. During his first post-doc at the Max Planck Institute for Biology of Ageing in Cologne, Germany, he worked on the doRiNA database of RNA interactions in post-transcriptional regulation. Returning to natural products research, Kai joined the Novo Nordisk Foundation Center for Biosustainability (NNFCFB) at the Technical University of Denmark, in Lyngby, Denmark. In addition to returning to antiSMASH development, he has driven the development of the antiSMASH database, the CRISPR/Cas sgRNA design tool CRISPy-web, and a number of smaller software projects. Kai is currently working as a Researcher at the NNFCFB, where he heads the bioinformatics group of the New Bioactive Compounds section ran by Sang Yup Lee and Tilmann Weber.</p>
                    </div>
                    
                    <button class="btn btn-primary" type="button" 
                        data-toggle="collapse" data-target="#keynote_2" 
                        aria-expanded="false" aria-controls="keynote_2">
                        Show Bio
                    </button>
                  </div>
				</div>

                <div class="col-sm-3">
                  <div class="pypharma-member">
					<img src="img/exec/michal_januszewski.jpg" alt="Michał Januszewski" class="img-circle pypharma-thumbnail">
					<h4>Michał Januszewski</h4>
					<h5>Google Research, Zurich</h5>

					<div id="keynote_3" class="collapse pypharma-bio">
					    <p>Michał Januszewski is a Staff Software Engineer at Google Research in Zürich, where he currently works on automated methods for high-throughput synaptic-resolution brain mapping. Prior to Google, Michał did research in the field of Computational Fluid Dynamics. He holds a PhD in Physics from University of Silesia in Katowice, Poland.</p>
                    </div>
                    
                    <button class="btn btn-primary" type="button" 
                        data-toggle="collapse" data-target="#keynote_3" 
                        aria-expanded="false" aria-controls="keynote_3">
                        Show Bio
                    </button>
                  </div>
				</div>

                <div class="col-sm-3">
                  <div class="pypharma-member">
					<img src="img/exec/greg_landrum.jpeg" alt="Greg Landrum" class="img-circle pypharma-thumbnail">
					<h4>Greg Landrum</h4>
					<h5>T5 Informatics</h5>
                    
					<div id="keynote_4" class="collapse pypharma-bio">
					    <p>After getting a Ph.D. in Theoretical Chemistry in the group of Roald Hoffmann at Cornell University and doing a post-doctoral fellowship in Aachen Germany Greg moved to California where he worked at a couple of startups and started applying machine learning to life-sciences problems. In 2006 he moved to Basel to work at the Novartis Insitutes for BioMedical Research. Greg spent about five years in the computer aided drug design group and then shifted to research IT to head the group responsible for chemistry software and data systems. Eventually he was also responsible for an internal initiative to integrate internal and external biological and chemical data and make it available to researchers. He now splits his time between KNIME and T5 Informatics, a consulting company providing support and services related to the open-source cheminformatics toolkit RDKit. Greg has been using Python together with C++ to solve scientific problems for 20 years now.</p>
                    </div>
                    
                    <button class="btn btn-primary" type="button" 
                        data-toggle="collapse" data-target="#keynote_4" 
                        aria-expanded="false" aria-controls="keynote_4">
                        Show Bio
                    </button>
                  </div>
				</div>
				
			</div>
		  
		 </div>
	  </section>

     <section id="workshops">
		<h2>Workshops</h2>
		  <div class="container">
			<div class="row">
				<div class="col-sm-12">
                    <table class="table" style="width:100%">
                      <tr>
                        <th>Title</th>
                        <th>Workshop Organizer</th>
                        <th>Affiliation</th>
                      </tr>
                      <tr>
                        <td>Deep Learning for Natural Language Processing in 
                        Pharma and Biomedical Applications</td>
                        <td>Diego Saldana</td>
                        <td>Roche</td>
                      </tr>
                      <tr>
                        <td>Genomic Data Analysis with Pandas</td>
                        <td>Maryan Zaheri and Carsten Magnus</td> 
                        <td>University of Zurich (UZH)</td>
                      </tr>
                      <tr>
                        <td>Bayesian Inference with Python</td>
                        <td>Elizaveta Semenova</td> 
                        <td>AstraZeneca</td>
                      </tr>
                      <tr>
                        <td><strike>How to Upscale Hyper-parameter tuning with Scikit learn using DASK on the cluster</strike></td>
                        <td><strike>Marius Rene Garmhausen</strike></td> 
                        <td><strike>Roche</strike></td>
                      </tr>
                      <tr>
                        <td>Tutorial on interpretability in machine learning for Computational Biology</td>
                        <td>An-Phi Nguyen</td>
                        <td>IBM</td>
                      </tr>
                      <tr>
                        <td>Cheminformatics workshop from QSAR to DNN using RDKit</td>
                        <td>David Marcus</td>
                        <td>GlaxoSmithKline</td>
                      </tr>
                      <tr>
                        <td>Snakemake for Reproducible Analyses</td>
                        <td>Romain Feron and Amina Echchiki</td> 
                        <td>University of Lausanne (UNIL)</td>
                      </tr>
                      <tr>
                        <td>Building pharmaceutical web applications with Dash</td>
                        <td>Rafal Chojnacki and Agata Figas</td> 
                        <td>Roche</td>
                      </tr>
                      <tr>
                        <td><strike>Machine learning methods for the analysis of genomic and structured data</strike></td>
                        <td><strike>Damian Roqueiro</strike></td> 
                        <td><strike>ETH</strike></td>
                      </tr>
                    </table>
                </div>
				
			</div>
		  
		 </div>
	  </section>

      <section id="executive-committee">
		<h2>Executive Committee</h2>
		  <div class="container">
			<div class="row">

                <div class="col-sm-3">
                  <div class="pypharma-member">
					<img src="img/exec/diego_saldana.jpeg" alt="Diego Saldana" class="img-circle pypharma-thumbnail">
					<h4>Diego Saldana</h4>
					<h5>Roche</h5>
                    
					<div id="exec_1" class="collapse pypharma-bio">
					    <p>Diego Saldana is a Data Scientist at Roche Personalized Healthcare (PHC). He has developed models to perform various tasks and analyze diverse data sources. Currently his main applications of interest are in oncology and clinico-genomics.</p>
                    </div>
                    
                    <button class="btn btn-primary" type="button" 
                        data-toggle="collapse" data-target="#exec_1" 
                        aria-expanded="false" aria-controls="exec_1">
                        Show Bio
                    </button>
                  </div>
				</div>

                <div class="col-sm-3">
                  <div class="pypharma-member">
					<img src="img/exec/otto_fajardo.jpg" alt="Otto Fajardo" class="img-circle pypharma-thumbnail">
					<h4>Otto Fajardo</h4>
					<h5>Roche</h5>
                    
					<div id="exec_2" class="collapse pypharma-bio">
					    <p>Otto Fajardo works at Roche in the Biometrics department handling Real World and Clinical Data. He uses python in conjunction with relational databases to perform his job. He authors packages for data handling, most of them internal to Roche but also open source (pyreadstat, pyreadr). He has a background in Neuroscience (PhD) and Dentistry and has been using Python everyday for the last 10 years.</p>	
                    </div>
                    
                    <button class="btn btn-primary" type="button" 
                        data-toggle="collapse" data-target="#exec_2" 
                        aria-expanded="false" aria-controls="exec_2">
                        Show Bio
                    </button>
                  </div>
				</div>
				
				<div class="col-sm-3">
                  <div class="pypharma-member">
					<img src="img/exec/carsten_magnus.jpg" alt="Carsten Magnus" class="img-circle pypharma-thumbnail">
					<h4>Carsten Magnus</h4>
					<h5>University of Zurich</h5>
                    
					<div id="exec_3" class="collapse pypharma-bio">
					    <p>Carsten Magnus is a theoretical and computational biologist at the Institute of Medical Virology, University of Zurich. In his research he develops models and methods (mainly implemented in R and python) to tackle important questions on HIV and Influenza evolution. Check out his webpage <a href="https://magnus-research.org">webpage</a> for further information.</p>	
                    </div>
                    
                    <button class="btn btn-primary" type="button" 
                        data-toggle="collapse" data-target="#exec_3" 
                        aria-expanded="false" aria-controls="exec_3">
                        Show Bio
                    </button>
                  </div>
				</div>
				
				<div class="col-sm-3">
                  <div class="pypharma-member">
					<img src="img/exec/maryam_zaheri.jpeg" alt="Maryam Zaheri" class="img-circle pypharma-thumbnail">
					<h4>Maryam Zaheri</h4>
					<h5>University of Zurich</h5>
                    
					<div id="exec_4" class="collapse pypharma-bio">
					    <p>Maryam Zaheri is a scientific assistant and computational biologist at the Institute of Medical virology, University of Zurich. She develops models and methods mainly in the field of metagenomics, drug resistant mutation detection and research related to HIV virus.</p>	
                    </div>
                    
                    <button class="btn btn-primary" type="button" 
                        data-toggle="collapse" data-target="#exec_4" 
                        aria-expanded="false" aria-controls="exec_4">
                        Show Bio
                    </button>
                  </div>
				</div>
				
	        </div>

            <div class="row">

                <div class="col-sm-3">
                  <div class="pypharma-member">
					<img src="img/exec/geoffrey_fucile.jpg" alt="Geoffrey Fucile" class="img-circle pypharma-thumbnail">
					<h4>Geoffrey Fucile</h4>
					<h5>University of Basel</h5>
                    
					<div id="exec_5" class="collapse pypharma-bio">
					    <p>Geoffrey is a computational biologist/bioinformatician with the <a href="https://www.sib.swiss">SIB Swiss Institute of Bioinformatics</a> and the <a href="https://www.unibas.ch">University of Basel</a>. He generally works with biologists engaged in scientific computing at the high-performance computing center (<a href="https://scicore.unibas.ch">sciCORE</a>), and with the broader research community in data science applications and training.</p>
                    </div>
                    
                    <button class="btn btn-primary" type="button" 
                        data-toggle="collapse" data-target="#exec_5" 
                        aria-expanded="false" aria-controls="exec_5">
                        Show Bio
                    </button>
                  </div>
				</div>

                <div class="col-sm-3">
                  <div class="pypharma-member">
					<img src="img/exec/elizaveta_semenova.jpeg" alt="Elizaveta Semenova" class="img-circle pypharma-thumbnail">
                    
					<h4>Elizaveta Semenova</h4>
					<h5>AstraZeneca</h5>
                    
					<div id="exec_6" class="collapse pypharma-bio">
					    <p>Elizaveta Semenova is a Post-doctoral Researcher at AstraZeneca working in Bayesian Machine Learning. She has a PhD in Epidemiology/Biostatistics with experience in spatio-temporal modelling, data analysis, theoretical and applied mathematics. Elizaveta is interested in Bayesian statistics, machine learning and probabilistic modelling. She is also a technological innovation and technical education enthusiast.</p>	
                    </div>
                    
                    <button class="btn btn-primary" type="button" 
                        data-toggle="collapse" data-target="#exec_6" 
                        aria-expanded="false" aria-controls="exec_6">
                        Show Bio
                    </button>
                  </div>
				</div>

                <div class="col-sm-3">
                  <div class="pypharma-member">
					<img src="img/exec/matteo_manica.jpg" alt="Matteo Manica" class="img-circle pypharma-thumbnail">
                    
					<h4>Matteo Manica</h4>
					<h5>IBM</h5>
                    
					<div id="exec_7" class="collapse pypharma-bio">
                        <p>Matteo is a Research Staff Member in Cognitive Health Care and Life Sciences at IBM Research Zürich. He's currently working on the development of multimodal deep learning models for drug discovery using chemical features and omic data. He also researches in multimodal learning techniques for the analysis of pediatric cancers in a H2020 EU project, iPC, with the aim of creating treatment models for patients. He received his degree in Mathematical Engineering from Politecnico di Milano in 2013. After getting his MSc he worked in a startup, Moxoff spa, as a software engineer and analyst for scientific computing. In 2019 he obtained his doctoral degree at the end of a joint PhD program between IBM Research and the Institute of Molecular Systems Biology, ETH Zürich, with a thesis on multimodal learning approaches for precision medicine.</p>	
                    </div>
                    
                    <button class="btn btn-primary" type="button" 
                        data-toggle="collapse" data-target="#exec_7" 
                        aria-expanded="false" aria-controls="exec_7">
                        Show Bio
                    </button>
                  </div>
				</div>
				
				<div class="col-sm-3">
                  <div class="pypharma-member">
					<img src="img/exec/an_phi_nguyen.jpg" alt="An-phi Nguyen" class="img-circle pypharma-thumbnail">
					<h4>An-phi Nguyen</h4>
					<h5>IBM</h5>
                    
					<div id="exec_8" class="collapse pypharma-bio">
					    <p>An-phi is a doctoral student in the Cognitive Health Care and Life Sciences group (IBM Research Zurich) and in the Seminar for Statistics of ETH Zurich. His current research mainly focuses on interpretability for machine learning models with applications to computational biology. From time to time he also wonders about other topics in Machine Learning, Statistics, NLP and Computer Vision.
                    
                    He received his BSc degree in Mathematical Engineering from Politecnico di Milano and his MSc degree in Computational Science and Engineering from ETH Zurich. Before joining IBM Research, he worked in two startups as algorithm/software engineer (Insightness AG, TrueAI Ltd.).</p>	
                    </div>
                    
                    <button class="btn btn-primary" type="button" 
                        data-toggle="collapse" data-target="#exec_8" 
                        aria-expanded="false" aria-controls="exec_8">
                        Show Bio
                    </button>
                  </div>
				</div>
				
	        </div>
            
            <div class="row">
            
                <div class="col-sm-3">
                  <div class="pypharma-member">
					<img src="img/exec/david_marcus.png" alt="David Marcus" class="img-circle pypharma-thumbnail">
                    
					<h4>David Marcus</h4>
					<h5>GlaxoSmithKline</h5>
                    
					<div id="exec_9" class="collapse pypharma-bio">
					    <p>I am a cheminformaticitan working at GlaxoSmithKline on various machine learning projects at data and computational sciences department, GSK R&amp;D Stevenage site, United Kingdom. My background is Pharmacy, Medicinal Chemistry and a PhD in Computational Chemistry at the Hebrew University of Jerusalem, followed up by two postdocs, at the University of Cambridge and at EMBL-EBI. My main research interests are small-molecule libraries selection and project design for hit and lead generation in which I devleop tools for compound generation and Active Learning. I am also co-leading the small-molecule predictive modelling centre of excellence within GSK overlooking models progression and development for small-molecule predictions, from QSAR to DNN models, including commercial platfrom as well as in-house proprietry data driven tools.</p>	
                    </div>
                    
                    <button class="btn btn-primary" type="button" 
                        data-toggle="collapse" data-target="#exec_9" 
                        aria-expanded="false" aria-controls="exec_9">
                        Show Bio
                    </button>
                  </div>
				</div>
                
                <div class="col-sm-3">
                  <div class="pypharma-member">
					<img src="img/exec/simon_dirmeier.jpg" alt="Simon Dirmeier" class="img-circle pypharma-thumbnail">
					<h4>Simon Dirmeier</h4>
					<h5>ETH</h5>
                    
					<div id="exec_10" class="collapse pypharma-bio">
					    <p>Simon Dirmeier is a doctoral student in the Computational Biology Group at D-BSSE, ETH Zurich. His scientific interests revolve mainly around graphical and probabilistic modelling of genetic perturbation screens. Furthermore, he works on scalable methods for analysing large-scale imaging data.</p>
                    </div>
                    
                    <button class="btn btn-primary" type="button" 
                        data-toggle="collapse" data-target="#exec_10" 
                        aria-expanded="false" aria-controls="exec_10">
                        Show Bio
                    </button>
                  </div>
				</div>
                
                <div class="col-sm-3">
                  <div class="pypharma-member">
					<img src="img/exec/damian_roqueiro.jpg" alt="Damian Roqueiro" class="img-circle pypharma-thumbnail">
					<h4>Damian Roqueiro</h4>
					<h5>ETH</h5>
					<div id="exec_11" class="collapse pypharma-bio">
					    <p>Damian is a Senior Researcher at the Machine Learning and Computational Biology Lab at ETH Zurich. He develops and implements machine learning models to extract knowledge from patients’ clinical and omics data. His research interests lie in the analysis of complex networks, with a current focus on using electrophysiological recordings to model the connectivity of neuronal cultures.</p>
                    </div>
                    
                    <button class="btn btn-primary" type="button" 
                        data-toggle="collapse" data-target="#exec_11" 
                        aria-expanded="false" aria-controls="exec_11">
                        Show Bio
                    </button>
                  </div>
				</div>

                <div class="col-sm-3">
                  <div class="pypharma-member">
					<img src="img/exec/placeholder.png" alt="Moritz Freidank" class="img-circle pypharma-thumbnail">
					<h4>Moritz Freidank</h4>
					<h5>Novartis</h5>
					<div id="exec_12" class="collapse pypharma-bio">
					    <p>This site is under construction.</p>
                    </div>
                    
                    <button class="btn btn-primary" type="button" 
                        data-toggle="collapse" data-target="#exec_12" 
                        aria-expanded="false" aria-controls="exec_12">
                        Show Bio
                    </button>
                  </div>
				</div>

            </div>
            
            <div class="row">
                
                <div class="col-sm-3">
                  <div class="pypharma-member">
					<img src="img/exec/placeholder.png" alt="Alex Perera" class="img-circle pypharma-thumbnail">
					<h4>Alex Perera</h4>
					<h5>Polytechnic University of Catalonia</h5>
					<div id="exec_13" class="collapse pypharma-bio">
					    <p>This site is under construction.</p>
                    </div>
                    
                    <button class="btn btn-primary" type="button" 
                        data-toggle="collapse" data-target="#exec_13" 
                        aria-expanded="false" aria-controls="exec_13">
                        Show Bio
                    </button>
                  </div>
				</div>
            

                <div class="col-sm-3">
                  <div class="pypharma-member">
					<img src="img/exec/placeholder.png" alt="Maria Jesus Algar" class="img-circle pypharma-thumbnail">
					<h4>Maria Jesus Algar</h4>
					<h5>Universidad de Alcala</h5>
                    
					<div id="exec_14" class="collapse pypharma-bio">
					    <p>This site is under construction.</p>
                    </div>
                    
                    <button class="btn btn-primary" type="button" 
                        data-toggle="collapse" data-target="#exec_14" 
                        aria-expanded="false" aria-controls="exec_14">
                        Show Bio
                    </button>
                  </div>
				</div>
                
            </div>
		  
		 </div>
	  </section>


      <section id="diversity-grants">
		<h2>Diversity Grants</h2>
		  <div class="container">
			<div class="row">
                <div class="col-sm-12">
                    <p>Roche/Genentech is offering to cover travel and accommodation expenses for two conference attendees to support the diversity in the open source Python healthcare community. The eligible candidates have to be currently enrolled as students, or work for a non-profit organization.</p>
                    <p>Update: Two candidates have been selected and applicants have now been informed of the panel's decision.</p>
                </div>
            </div>
		  
		 </div>
	  </section>
      
	  
	 <section id="contact">
        
		<h2>Contact Us</h2>
	  	<div class="container">
            <div class="row">
            
				<div class="col-sm-4">
		            <h3>E-mail</h3>
                    <p><a href="mailto:pythoninpharma@gmail.com">Get in touch</a> if you have questions or would like to learn more about PyPharma.</p>
                </div>
                
				<div class="col-sm-4">
		            <h3>Slack</h3>
                    <p>We have an active organizing community in slack with members of many companies and institutions in the pharmaceutical industry and academia. <a href="mailto:pythoninpharma@gmail.com">Contact us</a> if you're interested in joining our community.</p>
                </div>
                
				<div class="col-sm-4">
		            <h3>Mailing List</h3>
                    <p>Alternatively, you can sign up to our mailing list using the form below.</p>
		 	        <iframe width="540" height="305" src="https://sibforms.com/serve/MUIEAFvYLZxiOk8pH9fxKVPW_utGRIAtUdC3ZGWdpMR6I2CVbQOT0lMR9_9rkmW9Shfxdas5sDEZvcFREFeSjRDOLpezodCSeq9KDWVUlaRS6BR3EWcQxUWFnxzfkfLi1IZkdIA8gSMYFDsk15q8H-HzUI2-lnOJZ45P_Sjfubp1lpGWcjeIzvKj_eX-1dhCNXKNHVjjfkVMmhMW" frameborder="0" scrolling="auto" allowfullscreen style="display: block;margin-left: auto;margin-right: auto;max-width: 100%;"></iframe>
                </div>
                
            </div>
		</div>
	  </section> -->
	  
	  <footer>
	  		crafted with &hearts; in calgary by <a href="https://github.com/Ashwin7mak">ashwin</a>, background picture of <a href="https://commons.wikimedia.org/wiki/File:View_on_Basel_and_Middle_Bridge_from_M%C3%BCnsterplatz.jpg">Basel by Max Karmazin</a>
	  </footer>
	  
	  
    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
    <!-- Include all compiled plugins (below), or include individual files as needed -->
    <script src="bootstrap/js/bootstrap.min.js"></script>
  </body>
</html>
